src/app/login/login.component.scss

```
@@ -580,117 +185,192 @@
+.version-label {
+  font-weight: 500;
+  font-size: 0.75rem;
+  color: $text-muted;
+}
+
+.version-value {
+  font-weight: 400;
+  font-size: 0.75rem;
+  color: $text-dark;
+  text-align: right;
+  word-break: break-word;
```

Stylelint (17.13.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)